package.json

```diff
@@ -116,4 +116,4 @@
       }
     }
   }
-}
```

Comment:
Irrelevant change, can you please remove it from the PR?

Reply:
👍

Reply:
done